modules/cco-ccoctl-upgrading.adoc

```diff
@@ -15,27 +16,26 @@ ifdef::aws-sts[]
 = Updating AWS resources with the Cloud Credential Operator utility
 
 The process for upgrading an {product-title} cluster configured for manual mode with AWS Secure Token Service (STS) is similar to installing on a cluster for which you create the AWS resources individually.
-endif::aws-sts[]
 
+[NOTE]
+====
+By default, `ccoctl` creates objects in the directory in which the commands are run. To specify a directory, use the `--output-dir` flag. This procedure uses `<path_to_ccoctl_output_dir>` to refer to this location.
+
+Some `ccoctl` commands make AWS API calls to create or modify AWS resources. To place JSON files on the local file system instead, use the `--dry-run` flag. These JSON files can be reviewed or modified and then applied with the AWS CLI tool using the `--cli-input-json` parameters.
+====
+endif::aws-sts[]
 ifdef::google-cloud-platform[]
 = Updating GCP resources with the Cloud Credential Operator utility
 
 The process for upgrading an {product-title} cluster configured for manual mode with GCP Workload Identity is similar to installing on a cluster for which you create the GCP resources individually.
-endif::google-cloud-platform[]
 
 [NOTE]
 ====
 By default, `ccoctl` creates objects in the directory in which the commands are run. To specify a directory, use the `--output-dir` flag. This procedure uses `<path_to_ccoctl_output_dir>` to refer to this location.
 
-ifdef::aws-sts[]
-Some `ccoctl` commands make AWS API calls to create or modify AWS resources.
-endif::aws-sts[]
-ifdef::google-cloud-platform[]
-Some `ccoctl` commands make GCP API calls to create or modify GCP resources.
-endif::google-cloud-platform[]
-To place JSON files on the local file system instead, use the `--dry-run` flag. These JSON files can be reviewed or modified and then applied with the AWS CLI tool using the `--cli-input-json` parameters.
+Some `ccoctl` commands make GCP API calls to create or modify GCP resources. To place JSON files on the local file system instead, use the `--dry-run` flag. These JSON files can be reviewed or modified and then applied with the AWS CLI tool using the `--cli-input-json` parameters.
```

Comment:
we do not create JSON files for GCP. Instead of that we create bash scripts with `gcloud cli` commands that user can review and run to create resources. The content of bash script is something similar to following
```
#!/bin/sh
gcloud iam workload-identity-pools create arane-01 --location=global --description="Created by OpenShift ccoctl" --display-name=arane-01
```

```suggestion
Some `ccoctl` commands make GCP API calls to create or modify GCP resources. To avoid making API calls and create a bash scripts with gcloud CLI commands on the local file system instead, use the `--dry-run` flag. These bash scripts can be reviewed or modified and then run to create the required GCP resources.
```